server/appengine/src/main/java/who/Client.java

```diff
@@ -0,0 +1,46 @@
+package who;
+
+import com.googlecode.objectify.annotation.Cache;
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.condition.IfNotNull;
+
+import static com.googlecode.objectify.ObjectifyService.ofy;
+
+/**
+ * Represents an app client.
+ */
+@Entity @Cache public class Client {
+
+  @Id String uuid;
```

Comment:
Can we consistently call this clientId here and in the header, or change the header to `Who-Client-UUID`?

Reply:
This is private and in the context of the Client class, so clientId would be redundant. Best to keep "ID" in the header in case we want to use a different type of ID down the road.